OSDOCS-RHDEVDOCS-4304: shared resource updates for validating admission webhooks (#56497)
openshift/openshift-docs: modules/ephemeral-storage-using-a-sharedsecrets-resource-in-a-pod.adoc

* You have created a `SharedSecret` CR instance for the secret you want to share across namespaces in the cluster.
* You must have permission to perform the following actions
** Discover which `SharedSecret` CR instances are available by entering the `oc get sharedsecrets` command and getting a non-empty list back.

> ** Determine if the service account your pod will specify is allowed to use the given `SharedSecret` CR instance. That is, you can run `oc adm policy who-can use <identifier of specific SharedSecret>` to see if the service account in your namespace is listed.

Reviewer: ```suggestion
** Determine if the service account your pod specifies is allowed to use the given `SharedSecret` CR instance. That is, you can run `oc adm policy who-can use <identifier of specific SharedSecret>` to see if the service account in your namespace is listed.
```

> ** Determine if the service account your pod will specify is allowed to use `csi` volumes, of if you as the requesting user (if you create the pod directly) is allowed to use `csi` volumes.  See "Understanding and managing pod security admission" for details.

Reviewer: Maybe:
```suggestion
** Determine if the service account your pod specifies is allowed to use `csi` volumes, or if you, as the requesting user who created the pod directly, are allowed to use `csi` volumes.  See "Understanding and managing pod security admission" for details.
```
